drivers/Watchdog.h

```diff
@@ -59,10 +64,6 @@ namespace mbed {
  * @note Synchronization level: Interrupt safe
  * @ingroup drivers
  */
-/**
```

Comment:
Line `@ingroup drivers` probably not harmful but unnecessary.

Reply:
It is a bit too late to make that comment as all drivers, rtos, and events files use this pattern.
I would keep it as is for consistency.

Reply:
This is inaccurate. `@ingroup drivers` was removed in most places where not longer required.
In fact I can only see it in this file and ResetReason.h which might have been missed from a recent rebase.

Reply:
It is there. 
Look: 
https://github.com/ARMmbed/mbed-os/blob/feature-public-headers/drivers/AnalogIn.h#L29
https://github.com/ARMmbed/mbed-os/blob/feature-public-headers/rtos/EventFlags.h#L33
https://github.com/ARMmbed/mbed-os/blob/feature-public-headers/events/Event.h#L24

Reply:
Remove the one you talked about here: https://github.com/ARMmbed/mbed-os/compare/af5f264e6b71d515737fcd201b543d8241c12a5d..a41f47eb0bb8c89dd796d433cfef8fafad920fd8